Makefile

```diff
@@ -85,3 +85,11 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+clean-api-reference:
+	rm -rf content/en/docs/reference/kubernetes-api/ref/
+
+api-reference: clean-api-reference ## Build the API reference pages. go needed
+	mkdir content/en/docs/reference/kubernetes-api/ref/
+	cd api-reference && \
+		go run cmd/main.go kwebsite --config-dir config/v1.19/ --file api/v1.19/swagger.json --output-dir ../content/en/docs/reference/kubernetes-api/ref/
```

Comment:
Might we want to include `make api-reference` in our Netlify build commands? 

@sftim @kbhawkey

Reply:
> 
> 
> Might we want to include `make api-reference` in our Netlify build commands?
> 
> @sftim @kbhawkey

Hi @zacharysarah ,
I am trying to understand why this target should be part of the automated build/deploy? How often will the
generated pages change?

Reply:
I have pushed a change to the README to explain the build process for a new Kubernetes release.

The idea is to rebuild the references pages once for each Kubernetes release, and to include the generated pages in the Git repo.

Reply:
@kbhawkey 👋 

> How often will the generated pages change?

Ideally, we'd regenerate the reference pages whenever the API spec changes in k/k, preferably without manual intervention.

Automation is _definitely_ future iteration territory, so I'm fine with dropping the question for now. It's certainly not a blocker!

Reply:
At the moment, I think the answer to this question is no.